docs/plugins/cli.md

```diff
@@ -35,10 +35,13 @@ NGXS CLI
 
   $ ngxs --name name --spec boolean --directory path --folderName name
   $ ngxs --help
+  
 Options
 
-  --name name         Store name  --directory path    By default, the prompt is set to the current directory
-  --folderName name   Use your own folder name, default: state  --spec boolean      Creates a spec file for store, default: true
+  --name name         Store name  
+  --directory path    By default, the prompt is set to the current directory
+  --folderName name   Use your own folder name, default: state  
```

Comment:
@splincode @kyusupov33 
Shouldn't the `--folderName` (camel case) argument or rather be `--folder-name` (snake case I think it is called)?
I think that makes it have better support and less ambiguity across operating systems... or at least follows the broader convention.
The `ng` cli has the following syntax for example (from our package.json):
`ng test --watch=false --code-coverage --progress=false --project ngxs`

Reply:
PS. Will have to update the cli demo gif to match the renamed param too ;-)

Reply:
done